css/styles.css

```diff
@@ -444,15 +444,15 @@ tr:nth-child(odd) {
 }
 .logo-link {
     display: inline-block;
-    width: 220px;
-    height: 64.77px;
-    background: url("/images/logo.png");
```

Comment:
Should we remove the old logo image file?

Reply:
removed